src/action.js

```diff
@@ -70,25 +66,47 @@ async function run() {
       repoOwner: owner.login,
     });
 
-    core.setOutput("linked_issues_count", linkedIssuesCount);
-    core.setOutput("issues", issues);
-
-    if (!linkedIssuesCount) {
-      const prId = pullRequest?.id;
-      const shouldComment =
-        !linkedIssuesComments.length && core.getBooleanInput("comment") && prId;
+    if (skipCheck) {
+      core.debug("Pull request is labeled with 'no-issue', skipping check");
 
-      if (shouldComment) {
-        const body = core.getInput("custom-body-comment");
-        await addComment({ octokit, prId, body });
+      if (linkedIssuesComments.length) {
+        await deleteLinkedIssueComments(octokit, linkedIssuesComments);
 
-        core.debug("Comment added");
+        core.debug(`${linkedIssuesComments.length} comment(s) deleted.`);
       }
 
-      core.setFailed(ERROR_MESSAGE);
-    } else if (linkedIssuesComments.length) {
-      await deleteLinkedIssueComments(octokit, linkedIssuesComments);
-      core.debug(`${linkedIssuesComments.length} Comment(s) deleted.`);
+      core.setOutput("check_skipped", true);
```

Comment:
can you please do an early return here, so that we don't need to have an unnecessary level of nesting of the code in the else branch?